core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -351,7 +352,9 @@ describe('Learner dashboard functionality', function() {
     await skillEditorPage.get(skillId);
 
     await skillEditorPage.moveToQuestionsTab();
-    await skillEditorPage.clickCreateQuestionButton();
+    for (var i = 1; i <= constants.TOPIC_MINIMUM_QUESTIONS_TO_PRACTICE; i++) {
+      await skillEditorPage.clickCreateQuestionButton();
+    }
```

Comment:
After we click the create question button, we need to fill out the entire page with the question details and save it. That's what the code from lines 358-376 do. So, we really want a for loop starting in the same place that you have it. But we want the bracket `}` to be after line 377 (after `moveToQuestionsTab`). Does that make sense?

Reply:
ok sorry I misunderstood... I'll fix it!